common/determined_common/experimental/determined.py

```diff
@@ -44,8 +44,8 @@ def get_checkpoint(self, uuid: str) -> Checkpoint:
         Get the :class:`~determined.experimental.Checkpoint` representing the
         checkpoint with the provided UUID.
         """
-        r = api.get(self._session._master, "checkpoints/{}".format(uuid)).json()
-        return Checkpoint.from_json(r, master=self._session._master)
+        r = api.get(self._session._master, "/api/v1/checkpoints/{}".format(uuid)).json()
+        return Checkpoint.from_json(r["checkpoint"], master=self._session._master)
```

Comment:
This is great, so long as you don't see any issues with it (I'm unfamiliar with these APIs).